openapi/openapi.yaml

```
@@ -0,0 +1,299 @@
+            type: string
+          expected_number_of_awards:
+            type: string
+        required:
+          - ceiling
+          - floor
+      CloseDate:
+        type: object
+        properties:
+          date:
+            type: string
+            format: date
+          explanation:
+            type: string
+      Email:
+        type: object
+        properties:
+          email:
+            type: string
+            format: email
+          description:
+            type: string
+      FundingActivityCategory:
+        type: object
+        properties:
+          name:
+            type: string
+            enum:
+              - 'Recovery Act'
+              - 'Agriculture'
+              - 'Arts'
+              - 'Business and Commerce'
+              - 'Community Development'
+              - 'Consumer Protection'
+              - 'Disaster Prevention and Relief'
+              - 'Education'
+              - 'Employment, Labor and Training'
+              - 'Energy'
+              - 'Environment'
+              - 'Food and Nutrition'
+              - 'Health'
+              - 'Housing'
+              - 'Humanities'
+              - 'Information and Statistics'
+              - 'Income Security and Social Services'
+              - 'Law, Justice and Legal Services'
+              - 'Natural Resources'
+              - 'Other'
+              - 'Opportunity Zone Benefits'
+              - 'Regional Development'
+              - 'Science and Technology and Other Research and Development'
+              - 'Transportation'
+              - 'Affordable Care Act'
+          code:
+            type: string
+            enum:
+              - 'RA'
+              - 'AG'
+              - 'AR'
+              - 'BC'
+              - 'CD'
+              - 'CP'
+              - 'DPR'
+              - 'ED'
+              - 'ELT'
+              - 'EN'
+              - 'ENV'
+              - 'FN'
+              - 'HL'
+              - 'HO'
+              - 'HU'
+              - 'IS'
+              - 'ISS'
+              - 'LJL'
+              - 'NR'
+              - 'O'
+              - 'OZ'
+              - 'RD'
+              - 'ST'
+              - 'T'
+              - 'ACA'
+      FundingActivity:
+        type: object
+        properties:
+          categories:
+            type: array
+            items:
+              $ref: '#/components/schemas/FundingActivityCategory'
+            minItems: 1
+          explanation:
+            type: string
+            nullable: true
+      FundingInstrument:
+        type: object
+        properties:
+          name:
+            type: string
+            enum: [
+              'Cooperative Agreement',
+              'Grant',
+              'Procurement Contract',
+              'Other',
+            ]
+          code:
+            type: string
+            enum: [
+              'CA',
+              'G',
+              'PC',
+              'O',
+            ]
```

Minor nit for consistency:

```suggestion
          name:
            type: string
            enum:
              - 'Cooperative Agreement'
              - 'Grant'
              - 'Procurement Contract'
              - 'Other'
          code:
            type: string
            enum:
              - 'CA'
              - 'G'
              - 'PC'
              - 'O'
```